src/Ontology/Asset.h

```diff
@@ -0,0 +1,33 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include <array>
+#include <vector>
+#include <string>
+
+#include "Account.h"
+#include "Transaction.h"
+#include "BinaryCoding.h"
+
+namespace TW {
+namespace Ontology {
+
+class Asset {
+protected:
+
+    const uint8_t txType = 0xD1;
+
+public:
+    virtual std::vector<uint8_t> contractAddress() = 0;
```

Comment:
You can use `Data` in (`Data.h`) instead of `std::vector<uint8_t>`

Reply:
OK, I will replace it.